tests/lsstseries_tests/test_analysis.py

```diff
@@ -40,21 +42,21 @@ def test_sf2_timeseries(lc_id):
         "flux_err": test_yerr,
         "band": ["r"] * len(test_y),
     }
-    ts = timeseries()
-    ts.meta['id'] = lc_id
-    test_ts = ts.from_dict(data_dict=test_dict)
-    res = test_ts.sf2(sthresh=100)
+    timseries = timeseries()
```

Comment:
small thing, but I think my preference is the "ts" shorthand over "timseries", more akin to pandas running with "df" as its default shorthand for a dataframe. It doesn't matter for these tests really, but in documentation we'll most likely be assigning this as "ts".

Reply:
Reverted.

pylint was complaining about short variable names, but repo convention trumps pylint!